.circleci/config.yml

```diff
@@ -1,11 +1,15 @@
 # TODO(mattklein123): See if there is a way to put build SHA into a variable.
 # In the meantime, it's required to edit the SHA in this file as well as envoy_build_sha.sh.
 
+references:
```

Comment:
Actually sorry can you delete the TODO comments above. Both of them are no longer needed!

Reply:
Sure thing.